universe/marathon.json.mustache

```diff
@@ -146,6 +146,41 @@
 ,"CASSANDRA_HINTS_FLUSH_PERIOD_IN_MS":"{{cassandra.hints_flush_period_in_ms}}"
 ,"CASSANDRA_MATERIALIZED_VIEW_WRITES":"{{cassandra.concurrent_materialized_view_writes}}"
 ,"CASSANDRA_COMMITLOG_TOTAL_SPACE_IN_MB":"{{cassandra.commitlog_total_space_in_mb}}"
+,"CASSANDRA_ROLES_UPDATE_INTERVAL_IN_MS":"{{cassandra.roles_update_interval_in_ms}}"
+,"CASSANDRA_PERMISSIONS_UPDATE_INTERVAL_IN_MS":"{{cassandra.permissions_update_interval_in_ms}}"
+,"CASSANDRA_KEY_CACHE_KEYS_TO_SAVE":"{{cassandra.key_cache_keys_to_save}}"
+,"CASSANDRA_ROW_CACHE_KEYS_TO_SAVE":"{{cassandra.row_cache_keys_to_save}}"
+,"CASSANDRA_COUNTER_CACHE_KEYS_TO_SAVE":"{{cassandra.counter_cache_keys_to_save}}"
+,"CASSANDRA_FILE_CACHE_SIZE_IN_MB":"{{cassandra.file_cache_size_in_mb}}"
+,"CASSANDRA_MEMTABLE_HEAP_SPACE_IN_MB":"{{cassandra.memtable_heap_space_in_mb}}"
+,"CASSANDRA_MEMTABLE_OFFHEAP_SPACE_IN_MB":"{{cassandra.memtable_offheap_space_in_mb}}"
+,"CASSAND,"RA_MEMTABLE_CLEANUP_THRESHOLD":"{{cassandra.memtable_cleanup_threshold}}"
```

Comment:
@zhiyanshao I don't think we should be modifying this file at all. The agreement with @triclambert and @gabrielhartmann was that it was fine to expose them as env variables that we at Uber can override, but Mesosphere does not want to expose all these parameters in the UI and complicate the configuration for DC/OS users.